Xamarin.Forms.Platform.iOS/Platform.cs

```diff
@@ -478,6 +479,30 @@ static void PresentPopUp(UIWindow window, UIAlertController alert, ActionSheetAr
 
 		internal class DefaultRenderer : VisualElementRenderer<VisualElement>
 		{
+			public override UIView HitTest(CGPoint point, UIEvent uievent)
+			{
+				// UIview hit testing ignores objects which have an alpha of less than 0.01 
+				// (see https://developer.apple.com/reference/uikit/uiview/1622469-hittest)
+				// To prevent layouts with low opacity from being implicitly input transparent, 
+				// we need to temporarily bump their alpha value during the actual hit testing,
+				// then restore it. If the opacity is high enough or user interaction is disabled, 
+				// we don't have to worry about it.
```

Comment:
Hi,
Great that issue 25943 has a fix on its way. I just want to make sure I understand the impact of it.
If one would have used `View.Opacity=0.0` instead of `View.Enabled = false` to disable a view, would this change the behavior from being input-transparent to handling the input?
I agree this is a misuse of Opacity, I just wanted to be sure I understood the fix.
Would it make sense to modify Alpha only if `0 < old <= 0.01` ?